data/basics/overflow.go

```diff
@@ -147,7 +105,7 @@ func (t *OverflowTracker) Sub(a uint64, b uint64) uint64 {
 }
 
 // Mul multiplies b from a with overflow detection
```

Comment:
Let's fix the comment while we're here: "Mul multiplies b by a with overflow detection"

Reply:
Fixed in [Revert "genericify OverflowTracker"](https://github.com/algorand/go-algorand/pull/5501/commits/40e2db1a3c9b3d3ce6faca39aa0fb73a2e604793)